src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/RawlsDAO.scala

```diff
@@ -89,7 +90,7 @@ trait RawlsDAO extends LazyLogging with ReportsSubsystemStatus {
 
   def patchCatalog(workspaceNamespace: String, workspaceName: String, updates: Seq[WorkspaceCatalog])(implicit userToken: WithAccessToken): Future[WorkspaceCatalogUpdateResponseList]
 
-  def getMethodConfigs(workspaceNamespace: String, workspaceName: String)(implicit userToken: WithAccessToken): Future[Seq[MethodConfigurationShort]]
+  def getAgoraMethodConfigs(workspaceNamespace: String, workspaceName: String)(implicit userToken: WithAccessToken): Future[Seq[AgoraConfigurationShort]]
```

Comment:
I chose `Future[Seq[AgoraConfigurationShort]]` as the return type based on the following:

If we ask Rawls for Agora configs, everything we get back should look like an Agora config. If that's not the case, it's better for the code to blow up immediately during JSON unmarshalling instead of potentially letting instances of the wrong thing float around in memory and try to put checks in place everywhere they could cause problems